src/excalidraw-app/index.tsx

```diff
@@ -273,7 +273,10 @@ const PlusLinkJSX = (
 
 const ExcalidrawWrapper = () => {
   const [errorMessage, setErrorMessage] = useState("");
-  const currentLangCode = languageDetector.detect() || defaultLang.code;
+  let currentLangCode = languageDetector.detect() || defaultLang.code;
+  if (typeof currentLangCode === "object") {
```

Comment:
```suggestion
  if (Array.isArray(currentLangCode)) {
```

if this should be an array, let's make the type check clear